_includes/install_instructions/shell.html

```diff
@@ -26,8 +26,11 @@ <h3>The Bash Shell</h3>
               </li>
               <li>
                 <strong>
-                  From the dropdown menu select "Use the Nano editor by default" (NOTE: you will need to scroll <emph>up</emph> to find it) and click on "Next".
+                  From the dropdown menu select "Use the nano editor by default" (NOTE: you may need to scroll <em>up</em> to find it) and click on "Next".
```

Comment:
This reverts some of the changes in #711 (Nano -> nano and will -> may). Was this intentional?